satpy/readers/fci_l1c_fdhsi.py

```diff
@@ -149,7 +149,9 @@ def get_channel_dataset(self, channel):
         return group, root_group
 
     def calc_area_extent(self, key):
-        """Calculate area extent for a dataset."""
+        """Calculate area extent for a dataset.
+
+        C"""
         # Calculate the area extent of the swath based on start line and column
         # information, total number of segments and channel resolution
         # numbers from Package Description, Table 8
```

Comment:
These values numbers can and therefore should be taken form the file and dimensions, with the assumption that the y-dimension equals the x-dimension (full disk image is always square).